Edit an uploaded file causes server error

Click edit button on a model:

<img width="1301" height="632" alt="Image" src="https://github.com/user-attachments/assets/3567cc10-ef8a-4473-9316-c91402a660d0" />

results in error:

<img width="522" height="310" alt="Image" src="https://github.com/user-attachments/assets/bf77ac23-9107-4510-8a9a-cb0c041061d8" />

it does not matter what data i enter or change. the error occurs also if no change is done and we click apply.

**debug log from container:**
```
app-1              | 10:14:12 rails.1              | [8c0b0171-5dc0-4650-ba49-0e1ef536b99a] {"method":"PATCH","path":"/models/d2sgv5ld15cp/model_files/djpj50xk08hd","format":"turbo_stream","controller":"ModelFilesController","action":"update","status":500,"allocations":3789,"duration":16.51,"view":0.0,"db":4.9,"@timestamp":"2026-05-08T10:14:12.547Z","@version":"1","message":"[500] PATCH /models/d2sgv5ld15cp/model_files/djpj50xk08hd (ModelFilesController#update)"}
app-1              | 10:14:12 rails.1              | [8c0b0171-5dc0-4650-ba49-0e1ef536b99a]   
app-1              | 10:14:12 rails.1              | [8c0b0171-5dc0-4650-ba49-0e1ef536b99a] ActiveModel::UnknownAttributeError (unknown attribute 'printed' for ModelFile.)
app-1              | 10:14:12 rails.1              | [8c0b0171-5dc0-4650-ba49-0e1ef536b99a] Caused by: NoMethodError (undefined method 'printed=' for an instance of ModelFile)
app-1              | 10:14:12 rails.1              | [8c0b0171-5dc0-4650-ba49-0e1ef536b99a] 
app-1              | 10:14:12 rails.1              | Information for: ActiveModel::UnknownAttributeError (unknown attribute 'printed' for ModelFile.):
app-1              | 10:14:12 rails.1              | [8c0b0171-5dc0-4650-ba49-0e1ef536b99a]   
app-1              | 10:14:12 rails.1              | [8c0b0171-5dc0-4650-ba49-0e1ef536b99a] app/controllers/model_files_controller.rb:85:in 'ModelFilesController#update'
app-1              | 10:14:12 rails.1              | [8c0b0171-5dc0-4650-ba49-0e1ef536b99a] app/controllers/application_controller.rb:118:in 'ApplicationController#switch_locale'
app-1              | 10:14:12 rails.1              | [8c0b0171-5dc0-4650-ba49-0e1ef536b99a] 

```

used version of Manyfold: v0.139.1

--- a/app/deserializers/form/model_file_deserializer.rb
+++ b/app/deserializers/form/model_file_deserializer.rb
@@ -5,7 +5,8 @@ module Form
       @params.require(:model_file).permit([
         :filename, # i18n-tasks-use t("activerecord.attributes.model_file.filename")
         :presupported, # i18n-tasks-use t("activerecord.attributes.model_file.presupported")
-        # printed isn't actually deserialized here # i18n-tasks-use t("activerecord.attributes.model_file.printed")
+        # printed isn't actually deserialized here but we'll include the form attribute anyway
+        # i18n-tasks-use t("activerecord.attributes.model_file.printed")
         :notes, # i18n-tasks-use t("activerecord.attributes.model_file.notes")
         :caption, # i18n-tasks-use t("activerecord.attributes.model_file.caption")
         :y_up, # i18n-tasks-use t("activerecord.attributes.model_file.y_up")

